docs/dao/code_of_conduct.md

```diff
@@ -0,0 +1,64 @@
+---
+id: code_of_conduct
+sidebar_label: Code of Conduct
+sidebar_position: 6
+slug: /dao/code_of_conduct
+---
+
+#  Code of Conduct for JAM Implementer's DAO
+
+_Note: This is only be taken as a draft and *NOT* considered final._   It is based off of the [Permanence DAO Code of Conduct](https://docs.permanence.io/code_of_conduct.html_).
```

Comment:
```suggestion
_Note: This is only be taken as a draft and *NOT* considered final._   It is based off of the [Permanence DAO Code of Conduct](https://docs.permanence.io/code_of_conduct.html).
```
Was there a newline meant to be? I see double space.

You can also use the under-construction warning from here, it formats with a yellow background

https://github.com/JamBrains/jam-docs/blob/8635418b9201e211be4604f0cae1052e50b15e3d/docs/advanced/rpc/index.md#L11-L13